gcp/extract/quantiles.py

```diff
@@ -33,7 +33,7 @@
 columns, basenames, transforms, vectransforms = configs.interpret_filenames(argv, config)
 
 # Collect all available results
-data = results.sum_into_data(config['results-root'], basenames, columns, config, transforms, vectransforms)
+data, years = results.sum_into_data(config['results-root'], basenames, columns, config, transforms, vectransforms)
```

Comment:
Also add `, years` to the other use of `sum_into_data` (line 41) or that will break.

Reply:
i'm assuming the `, parallel_deltamethod_years` fix above is preferred over `, years` in line 41

Reply:
Yes, thanks!